pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -696,7 +706,7 @@ private static int GetWaitCycles(int millis = WAIT_TIME)
                     var multipleViewProvider = tForm as IMultipleViewProvider;
                     if (multipleViewProvider != null)
                     {
-                        formType += "." + multipleViewProvider.ShowingFormView.GetType().Name;
+                        formType += "." + GetUIValue(() => multipleViewProvider.ShowingFormView.GetType().Name);
```

Comment:
A different way to solve this would be to move this line of code down two lines so it's inside of the "RunUI" that is already there.

Reply:
Hah, good point